src/utils/bar.js

```diff
@@ -75,6 +75,18 @@ export function addItem(bar, sourceItem, index) {
     return barItem;
 }
 
+/**
+ * Removes an item from its bar.
+ * @param {BarItem} barItem The bar item to remove.
+ * @param {BarDetails} [bar] The bar.
+ */
+export function removeItem(barItem, bar) {
+    if (!bar) {
```

Comment:
this is good but what do you think about export function( barItems,  bar = getItemsBar(barItems)){

bar.items = bar.items.filter(x  => x.id !== barItems.id);

}

Reply:
:+1: